Assignment/solution_student/css/style.css

```diff
@@ -1,27 +1,193 @@
 /*
 Project Name: HTML Template
 Client: Your Client
-Author: Your Name
+Author: Carly Brandess
 Developer: @poopsplat in Chicago
 */
 
-/******************************************
-/* BASE STYLES
+/* SETUP
 /*******************************************/
+/* Box Model Hack: http://www.paulirish.com/2012/box-sizing-border-box-ftw/ */
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+html {
+  box-sizing: border-box;
+}
+/* Micro clearfix hack: http://nicolasgallagher.com/micro-clearfix-hack/ */
+.clearfix {
+  *zoom: 1;
+}
+.clearfix:before,
+.clearfix:after {
+    content: " ";
+    display: table;
+}
+.clearfix:after {
+  clear: both;
+}
+
+a:hover {
+    text-decoration: none;
+}
+
+header { 
+ 	background-image: url("../images/header_bg.jpg");
+	width: 100%;
+	height: 500px;
+	background-size: contain;
+	background-repeat: no-repeat;
+	margin: 0;
+  	padding: 0;
+}
+
+h1 {
+	color: white;
+	font-weight: bolder;
+	font-size: 24px;
+	padding-left: 20px;
+	padding-top: 5px;
+}
+
+h2 {
+	color: white;
+	font-weight: semibold;
+	font-size: 14px;
+	border-radius: 5px;
+	text-align: center;
+	padding-top: 100px;
+}
+
+h3 {
+	color: #033048;
+	font-size: 20px;
+	font-weight: semibold;
+	padding: 0;
+	margin: 0;
+}
+
+.container {
+	padding: 0;
+	padding-bottom: 20px;
+	margin: 0;
+	width: 50%;
+	text-indent: 20px;
+}
+
+li {
+	text-indent: 20px;
+}
+
+.nowbutton {
+	text-align: center;
+}
+
+.now{
+	position: relative;
+	top: 75%;
+	padding: 10px 15px 11px;
+	font-size: 14px;
+	background-color: #f9e42e;
+	font-weight: bold;
+	color: #033048;
+	border-radius: 5px;
+	text-decoration: none;
+}
+
 body {
-  color: black;
-  font-size: 16px;
+  font-size: 13px;
+  font-weight: lighter;
   background: white;
+  padding: 50px;
+  color: #05080C;
+  line-height: 1.5;
+  max-width: 900px;
+  margin: 70px auto;
+  background: #F6F6F6;
+  border-top: 5px solid #222222;
+  font-family: 'open sans',sans-serif;
```

Comment:
This will only work if you have the `open sans` font installed on your machine since it's a non-system font, so it has to be included in the html from Google Web Fonts ([like this](https://www.google.com/fonts#QuickUsePlace:quickUse/Family:Open+Sans:400,300,300italic,400italic,600,600italic,700italic,700,800,800italic)) in the `head` of the html (see my comment there).